packages/server/postgres/queries/src/upsertIntegrationProviderQuery.sql

```diff
@@ -29,6 +33,8 @@ ON CONFLICT ("teamId", "service", "authStrategy") DO UPDATE SET
   "scope" = EXCLUDED."scope",
   "clientId" = EXCLUDED."clientId",
   "clientSecret" = EXCLUDED."clientSecret",
+  "consumerKey" = EXCLUDED."consumerKey",
```

Comment:
+1 
Names are hard! I leave the following to your discretion:

Oauth2 calls these `{clientId, clientSecret}`
Oauth1 calls these `{consumerKey, consumerSecret}` https://oauth.net/core/1.0a/
Jira Server calls these `{consumerKey, privateKey}`

As is, we're using Jira's terminology for all oauth1 stuff, which is probably OK since I hope it's the only time we'll ever use Oauth1.

However, if we wanted to reduce the number of columns in this table, we could share Oauth1 & Oauth2 fields. We'd have to change clientSecret to type `TEXT` (or a larger varchar), but that'd be it.

Reply:
I feel the names are confusing enough and I don't want to set a precedence for reusing columns for different data (although it's the same here, the different naming conventions ruin it). I will rename `privateKey` to `consumerSecret` though to keep the OAuth1 spirit.